src/App.js

```diff
@@ -33,14 +58,36 @@ class App extends Component {
                 </Main>
                 <FooterMenu />
                 <FooterTraditional/>
+                { notEntitled && <Modal
+                    title={ 'You are no entitled to use this application' }
```

Comment:
This should probably read `'You are not entitled to use this application'`

Reply:
Should be fixed now.